src/rendervulkan.cpp

```diff
@@ -177,6 +178,9 @@ struct PipelineInfo_t
 	uint32_t colorspaceMask;
 	bool st2084Output;
 	bool forceWideGammut;
+	bool enableItm;
+	float itmSdrNits;
+	float itmTargetNits;
```

Comment:
floats shouldn't go into the `PipelineInfo`, which is baked in at compile time, they should go into the FrameInfo_t which is a push constant.

Reply:
I see you fixed this up later, better to squash these together.

Reply:
yeah
guess I'll squash the whole commit